### PR TITLE
Add Beaufort windspeed to sensor markdown

### PR DESCRIPTION
**Pull request by fwestenberg**
## Proposed change

Add Beaufort windspeed scale to the sensor markdown

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/105795
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Comment by emontnemery** [comment]
Reopening as parent PR has been reopened

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/sensor.markdown [hunk 1]
```diff
@@ -69,7 +69,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **volume_storage**: Generic stored volume in L, mL, gal, fl. oz., m³, ft³, or CCF
 - **water**: Water consumption in L, gal, m³, ft³, or CCF
 - **weight**: Generic mass in kg, g, mg, µg, oz, lb, or st
-- **wind_speed**: Wind speed in ft/s, km/h, kn, m/s, or mph
+- **wind_speed**: Wind speed in Bft, ft/s, km/h, kn, m/s, or mph
```
Comment (emontnemery) [inline source/_integrations/sensor.markdown]: Please add Beaufort also to the `speed` device class

Comment (emontnemery) [inline source/_integrations/sensor.markdown]: Is it correct to shorten Beaufort to `Bft`?

**Review by emontnemery — CHANGES_REQUESTED** [review]
The number documentation should also be adjusted

File: source/_integrations/sensor.markdown (hunk 1 above)
Comment (fwestenberg) [reply source/_integrations/sensor.markdown]: Yes it is. It's someone's last name. :)

Comment (emontnemery) [reply source/_integrations/sensor.markdown]: I didn't mean it should not be capitalized, I meant to ask if `Bft` is an accepted abbreviation. Based on the discussion in the implementation PR https://github.com/home-assistant/core/pull/105795, I don't think it is and I think we should use the full name.

Comment (emontnemery) [inline source/_integrations/sensor.markdown]:
```suggestion
- **wind_speed**: Wind speed in Beaufort, ft/s, km/h, kn, m/s, or mph
```

**Review by emontnemery — DISMISSED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Review by frenck — APPROVED** [review]
Thanks, @fwestenberg 👍 

../Frenck

**Pull request merged**